mautic/mautic
# [UX] Add a tooltip on "How often to engage?" to explain what means to select "Once per session"

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
This enhancement adds a tooltip to the “How often to engage?” option, specifically for the “Once per Session” choice. It provides users with a clear explanation that selecting this option will trigger the engagement action only once during a visitor’s session, with a session defined as the period of active interaction until the visitor leaves the website. This tooltip is a valuable addition for users, helping them understand how to tailor engagement based on visitor behavior, thereby optimizing the user experience on their sites.




#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Focus Items
3. Create a new
4. Enter the Builder
5. Click on What the focus should be? > Display a notice
6. Hover over How often to engage

![image](https://github.com/mautic/mautic/assets/116097999/92fb6ce8-cbca-4901-8eee-32f8815b4032)
